src/tests/playwright/pages/realtime-sine.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script defer type="module">
+      import {record} from './util/recorder/recorder-main.js';
+      import {beCloseTo, test, evaluateTest} from './util/audit.js';
+
+      test((async () => {
+        const context = new AudioContext({sampleRate: 48000});
+        const oscillator = new OscillatorNode(context);
+        const duration = 1; // second
+        const {recorder, recordingCompletePromise} = await record(context, duration);
+        oscillator.connect(recorder).connect(context.destination);
+
+        oscillator.start();
+
+        const recordedChannelBuffers = await recordingCompletePromise;
+        const float32Buffer = recordedChannelBuffers[0];
+
+        await context.close();
+        return {buffer: float32Buffer};
+      })());
+
+      evaluateTest(async testResult => {
+        const bufferData = (await testResult).buffer;
+        const myRefData = await (await fetch('./reference/440@48k-sine-octx.json')).json();
+
+        // compare bufferData samples to reference
+        let numberOfAcceptableSamples = 0;
+        for (let i = 0; i < bufferData.length; i++) {
+          numberOfAcceptableSamples += beCloseTo(bufferData[i], myRefData[i], 0.01) ? 1 : 0;
+        }
+        console.info('% match:', numberOfAcceptableSamples / bufferData.length);
+
+        // return true if 99.8% pass
```

Comment:
Why 99.8%?

Also can we have this as a "constant named variable" up top? Something like `expectedAccuracy = 0.998`;